src/sentry/conf/server.py

```diff
@@ -1377,7 +1377,18 @@ def create_partitioned_queues(name):
     "redis": {
         "image": "redis:5.0-alpine",
         "ports": {"6379/tcp": 6379},
-        "command": ["redis-server", "--appendonly", "yes"],
+        "command": [
+            "redis-server",
+            "--appendonly",
+            "yes",
+            "--save",
+            "60",
+            "100",
```

Comment:
This will only save if we have 100 changed keys in a 60 second period, will that be enough to trigger it?

Reply:
We can make it a smaller number of keys like 10 or 20.